asm/rom_header.s

```diff
@@ -35,14 +35,35 @@
     .ascii "E"                  /* NTSC-U (North America) */
 #endif
 
+/* Savetype, region, and RTC */
 #if defined(SRAM)
-    .byte  0x32                 /* Version */
+    #if defined(USE_RTC)
```

Comment:
could these just be an OR instead of a bunch of if elses?

Reply:
While I don't like the conditional repetition here, I don't see a cleaner way of handling it without defining a macro or setting a variable or something.

Reply:
yeah thats what i was thinking of. just set the var to 0 if not defined and 1 if defined